### PR TITLE
Update ngscheckmate

### PR DESCRIPTION
**Pull request by cruk-mi**
----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by cruk-mi** [comment]
@BiocondaBot please add label

File: recipes/ngscheckmate/meta.yaml [hunk 1]
```diff
@@ -1,33 +1,40 @@
...
+  number: 0
   noarch: generic
 
 requirements:
   host:
-    - python 2.7.18
-    - R 4.1
-    - samtools 1.14
+    - python 2.*
+    - R 4.0
```
Comment (bgruening) [inline recipes/ngscheckmate/meta.yaml:16]: this should be r-base ... is the pin to version 4.0 really needed?

File: recipes/ngscheckmate/meta.yaml [hunk 2]
```diff
@@ -1,33 +1,40 @@
...
-  number: 4
+  number: 0
   noarch: generic
 
 requirements:
   host:
-    - python 2.7.18
-    - R 4.1
-    - samtools 1.14
+    - python 2.*
```
Comment (bgruening) [inline recipes/ngscheckmate/meta.yaml:15]: Python 2 is really out of date and is not supported anymore. Is this package still maintained?

File: recipes/ngscheckmate/meta.yaml (hunk 1 above)
Comment (cruk-mi) [reply recipes/ngscheckmate/meta.yaml]: I've now changed this to r-base and relaxed the version requirements

File: recipes/ngscheckmate/meta.yaml (hunk 2 above)
Comment (cruk-mi) [reply recipes/ngscheckmate/meta.yaml:15]: I can't speak for the developers but the package does not seem to be regularly updated. It happens to be in python 2 which is old, but the package is still quite useful.

Comment (SPPearce) [reply recipes/ngscheckmate/meta.yaml:15]: Yes, the package is really useful, but unfortunately is basically abandonware. I'm using it currently in Nextflow pipelines, and I'm after an updated container.

**Comment by dpryan79** [comment]
@BiocondaBot please fetch artifacts

**Review by dpryan79 — APPROVED** [review]

**Comment by cruk-mi** [comment]
Hi thanks for approving these changes. I can see that the merge commit is failing checks (there is a red cross attached to it). Not sure how to proceed as I can't tell why the commit is causing the checks to fail.

**Pull request merged**